### PR TITLE
Allow to switch to 11-11-10 format for cubemap probes

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
This helps saving on memory and make sampling them cheaper when 11-11-10 is selected. 

Given that they are not pre-exposed I didn't make it default to 11-11-10, but we should discuss this.

### CODE REVIEW

**Review by fredericv-unity3d — APPROVED** [review]

**Comment by FrancescoC-unity** [comment]
Update so that it is used for planar too, I expect tests too need changing, so I am running yamato.

**Comment by sebastienlagarde** [comment]
Any update on this one?

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/GlobalLightLoopSettings.cs [hunk 1]
```diff
@@ -27,6 +27,18 @@ public enum CubeReflectionResolution
         CubeReflectionResolution4096 = 4096
     }
 
+    /// <summary>
+    /// Available graphic formats for the cube reflection probes.
+    /// </summary>
+    [System.Serializable]
+    public enum ReflectionProbeFormat
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/GlobalLightLoopSettings.cs:34]: ReflectionProbeFormat
We need to say ReflectionAndPlanarProbeFormat as it apply to both, should be explicit in the enum and the code

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.Skin.cs [hunk 2]
```diff
@@ -153,6 +153,7 @@ public class GeneralSection
             public static readonly GUIContent compressProbeCacheContent = EditorGUIUtility.TrTextContent("Compress Reflection Probe Cache", "When enabled, HDRP compresses the Reflection Probe cache to save disk space.");
             public static readonly GUIContent cubemapSizeContent = EditorGUIUtility.TrTextContent("Reflection Cubemap Size", "Specifies the maximum resolution of the individual Reflection Probe cube maps.");
             public static readonly GUIContent probeCacheSizeContent = EditorGUIUtility.TrTextContent("Probe Cache Size", "Sets the maximum size of the Probe Cache.");
+            public static readonly GUIContent reflectionProbeFormatContent = EditorGUIUtility.TrTextContent("Reflection Probes Format", "Color format used for reflection probes. Keep in mind that probes are not pre-exposed when selecting the format.");
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.Skin.cs:156]: update: reflection probe and planar

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs [hunk 3]
```diff
@@ -258,6 +258,7 @@ static void Drawer_SectionReflection(SerializedHDRenderPipelineAsset serialized,
             }
 
             EditorGUILayout.Space();
+            EditorGUILayout.PropertyField(serialized.renderPipelineSettings.lightLoopSettings.reflectionProbeFormat, Styles.reflectionProbeFormatContent);
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs:261]: this also require to update the documentation (.md)

**Comment by FrancescoC-unity** [comment]
> Any update on this one?

All fixed but probably need a screenshot update. Will do now.

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**